src/components/Form/Step3/Step3.js

```diff
@@ -12,31 +12,18 @@ import SoMTicketNumber from './SoMTicketNumber/SoMTicketNumber';
 import UploadTicket from './UploadTicket/UploadTicket';
 import LastUsed from './LastUsed/LastUsed';
 import HowProcess from './HowProcess/HowProcess';
-import GenericError from '../../shared/Errors/GenericError';
-
```

Comment:
When i'm on this step, I can't seem to continue.
I keep getting this error when hitting continue (everything is valid):

![image](https://user-images.githubusercontent.com/13015201/102100735-afd8dc80-3e21-11eb-9a29-c8b32bc515a8.png)

Reply:
Confirmed as working